### PR TITLE
#4342 auto-compile both .js and .ts source code file

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4342)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
  - Only change json file
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
retry manager-e2e

File: ohara-manager/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "preinstall": "node ./scripts/checkYarnVersion.js",
     "setup": "yarn install --frozen-lockfile && yarn --cwd client setup",
-    "start": "cross-env NODE_ENV=development nodemon start.js --exec \"yarn lint:server && node\"",
+    "start": "cross-env NODE_ENV=development nodemon start.js --watch 'client/*' --exec \"yarn lint:server && ts-node\"",
```
Comment (konekoya) [inline ohara-manager/package.json]: So this excludes watching file changes made in ohara-manager, right?

File: ohara-manager/package.json [hunk 2]
```diff
@@ -54,6 +54,8 @@
     "lodash": "4.17.15",
     "morgan": "1.9.1",
     "sha.js": "2.4.11",
+    "ts-node": "8.6.2",
+    "typescript": "3.8.3",
```
Comment (konekoya) [inline ohara-manager/package.json]: Should we remove the typescript from client and just use this one for both ohara-manager and its client (UI)?

File: ohara-manager/package.json (hunk 1 above)
Comment (saivirtue) [reply ohara-manager/package.json]: Yep. We should watch client files only, right?

File: ohara-manager/package.json (hunk 2 above)
Comment (saivirtue) [reply ohara-manager/package.json]: Good point. Thanks for pointing out.

File: ohara-manager/package.json (hunk 1 above)
Comment (konekoya) [reply ohara-manager/package.json]: It'd be great if we can watch both :)
It's actually not a big deal for now as we don't change ohara-manager itself too often. But if we need to work on ohara-manager someday without the ability to watching the file changes and reload, we will need to manually kill it and restart whenever a change is made...

File: ohara-manager/package.json (hunk 2 above)
Comment (saivirtue) [reply ohara-manager/package.json]: Weird...it seems nodejs couldn't find the right path for us. I will add an issue to trace this in #4342

File: ohara-manager/package.json (hunk 1 above)
Comment (saivirtue) [reply ohara-manager/package.json]: Got it :)

**Pull request merged**